### DISCUSSION TITLE
Local tracks unexpectedly stop playing

### DISCUSSION THREAD

**Discussion by danyaridiger** [discordjs/discord.js#9213] [Q&A] [answered]
Hi all! I am trying to play local audio file by `createAudioResource` function, but after 20-30 seconds playing unexpectedly stops and repeated attempts to start the player lead to nothing. Here is my code fragment:

`
client.clientPlayer = createAudioPlayer();
`<br/>

`joinVoiceChannel({
`
&emsp;`
        channelId: newMember.channelId,
`
&emsp;`
        guildId: newMember.guild.id,
`
&emsp;`
        adapterCreator: newMember.guild.voiceAdapterCreator,
`
`
      }).subscribe(client.clientPlayer);
`
<br/>
`
      const track = createAudioResource("./static/track.mp3");
`
`await client.clientPlayer.play(track);`

I am using `0.14.0` version of `@discordjs/voice` package. 

Extra question: how to leave voice discord voice chat after local track playing and switch to other player (for expm. `discord-player`)?

**Comment by sdanialraza** [comment, marked as answer]
Hey, I think #9185 might be related to what you're talking about, have a look at that issue as there's a discussion on some hotfixes and more information about it, hope this was helpful!

**Reply by danyaridiger** [reply]
Thanks! In that issue i've found temporary solution for my problem.

**Reply by sdanialraza** [reply]
Of course, and glad to hear that.
